src/DynamoPackages/PackageManagerClient.cs

```diff
@@ -99,6 +99,23 @@ internal PackageHeader GetPackageHeader(IPackageInfo packageInfo)
             return header;
         }
 
+        /// <summary>
+        /// Gets maintainers for a specific package
+        /// </summary>
+        /// <param name="packageInfo"></param>
+        /// <returns></returns>
+        internal PackageHeader GetPackageMaintainers(IPackageInfo packageInfo)
+        {
+            var header = FailFunc.TryExecute(() =>
```

Comment:
this is a package header? Or is this a user name?

Reply:
part of the header, it will have all the maintainers.